packages/zod-openapi/README.md

```diff
@@ -284,6 +284,48 @@ const appRoutes = app.openapi(route, (c) => {
 const client = hc<typeof appRoutes>('http://localhost:8787/')
 ```
 
+## Tips
+
+### How to register components
+
+You can register components to the registry as follows:
```

Comment:
You have to add a link to:
https://github.com/asteasolutions/zod-to-openapi#defining-custom-components

Reply:
Done!

Reply:
Good!